### DISCUSSION TITLE
Support for the OKTA authentication into Strapi.

### DISCUSSION THREAD

**Discussion by divyansh0777** [strapi/strapi#7423] [Legacy Topics]
Hello Strapi team,

I hope you guys are safe and good. Actually I want to use OKTA authentication in the Strapi CMS so that only authenticated user can access the CMS. Is that possible for you guys to provide this support for okta and bypass the use of **strapi-admin** package?

**Comment by derrickmehaffy** [comment]
I need to confirm with the product team but I believe this feature: https://portal.productboard.com/strapi/1-public-roadmap/c/61-authentication-providers-sso

Should also cover the admin panel. I will try to get an update on this.

**Reply by divyansh0777** [reply]
Thank you so much @derrickmehaffy for letting us know. We'll wait for this update.

**Reply by derrickmehaffy** [reply]
Indeed I spoke with @Aurelsicoko and this feature is only for the Admin panel, users-permissions will be later.
